config_utilities/include/config_utilities/types/enum.h

```diff
@@ -196,9 +196,7 @@ struct Enum {
 // Simplified interfaces to include in 'declare_config'.
 
 /**
- * @brief Declare a field of a config to be an enum, that will be parsed and checked bystring  value names. Note that
```

Comment:
This comment is important and I'd suggest to keep it (it allows a config to override how to read enums compared to the global definition (if that exists))